### DISCUSSION TITLE
Impressora não fiscal 58mm

### DISCUSSION THREAD

**Discussion by Ajsgalvao** [RamonSilva20/mapos#2384] [General]
Seria possível ter alguma atualização para aquelas mini impressoras Bluetooth de 58mm mas que na realidade usa somente 48mm. Usam as bobinas de máquina de cartão de crédito, pois ao prestar o suporte ao cliente poderia imprimir diretamente pelo celular usando essa impressora

**Comment by johnteckinformatica** [comment]
Depois faça um teste, dentro de mapos\application\views\os 
imprimirOsTermica.php
Edite os campos de 72mm para 48mm e o 80mm para 58mm

**Reply by Ajsgalvao** [reply]
blz, vou fazer isso pra ver se diminui o tamanho e postarei aqui o resultado

**Reply by Ajsgalvao** [reply]
fiz os testes, o cabeçalho ele diminuiu mas o tamanho da descrição e dos produtos ficou do mesmo tamanho. Será que tem como alterar isso tb

**Reply by johnteckinformatica** [reply]
```
Faça uma copia por segurança do seu arquivo imprimirOsTermica.php

Edite essas linhas abaixo, você pode alterar o tamanho da fonte ao seu modo.

Edite essas linhas onde tem 11px e 12px e ponha 9px

<td style="width: 50%; padding-left: 0; font-size: 11px;">

<td style="text-align: center; width: 100%; font-size: 12px;">

<div style="margin-top: 0; padding-top: 0; font-size: 12px;">

<table style='font-size: 11px;' class="table table-bordered table-condensed" id="tblProdutos">

<table style='font-size: 11px;' class="table table-bordered table-condensed">

<table style='font-size: 11px;' class="table table-bordered table-condensed" id="tblProdutos">

<table style='font-size: 11px;' class="table table-bordered table-condensed">


Depois pesquise no arquivo, onde contenha 13px e ponha todos para 11px

Veja se melhora em algo.

**Reply by Ajsgalvao** [reply]
vou fazer e testar, obrigado por responder,  assim que editar volto com a resposta
